ovs-with-p4-arch.rst

```diff
@@ -1,20 +1,21 @@
-Introduction
--------------------------------------------------------------------------------
+Introduction:
 -------------------------------------------------------------------------------
+
 P4-OVS is a a fork of Open vSwitch to support programming offloads to 
 programmable pipelines via P4 language/specification. These patches will be 
 upstreamed back to the OVS community and are being hosted temporarily on 
 ipdk.io to get early feedback.
 
 Overview:
+--------------------------------------------------------------------------------
+
 P4 with OVS adds a new dataplane option (in addition to 'ofproto'), called 
 'p4proto' which connects up the kernel and OVS control planes into a P4 
 pipeline via the TDI (Table Driven Inteface) interface of IPDK.
 
 p4proto has the following functions:
 --------------------------------------------------------------------------------
-
-OVS Control Plane:  Interconnect the existing components of OVS such that a user 
+OVS Control Plane:  Interconnect the existing components of such that a user 
```

Comment:
These could be a list if you follow RST list structure as shown [here](https://bashtage.github.io/sphinx-material/rst-cheatsheet/rst-cheatsheet.html#lists).